docs/notebooks/survival_model.ipynb

```diff
@@ -0,0 +1,1587 @@
+{
```

Comment:
<p>It would be good to add something telling how <code>Surv</code> works in the model formula. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/bambi/pull/543/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>